packages/svelte/tests/suite.ts

```diff
@@ -76,13 +80,16 @@ async function for_each_dir<Test extends BaseTest>(
 	let created_test = false;
 
 	for (const dir of fs.readdirSync(`${cwd}/${samples_dir}`)) {
-		if (dir[0] === '.') continue;
+		if (dir[0] === '.' || !filter.test(dir)) continue;
 
-		const file = `${cwd}/${samples_dir}/${dir}/_config.js`;
-		if (!filter.test(dir)) {
+		// skip directories that don't have any files without _ prefix — these
+		// are leftover from a different branch, and will cause annoying errors
```

Comment:
This comment needs updating now, but I do feel more comfortable with this more conservative change.

Reply:
cool, i just removed it altogether since the other comment right above it serves the same purpose